apps/api/src/server/config/schema.js

```diff
@@ -28,5 +28,6 @@ export default joi.object({
 		levelFile: joi.string(),
 		levelStdOut: joi.string()
 	}),
-	cwd: joi.string()
+	cwd: joi.string(),
+	featureFlags: joi.object()
```

Comment:
Do we want to define here what keys we might expect in this object?

Reply:
The keys here can be any and as many as we need - is there a way to define this pattern in `joy`? they will all have a boolean as value

Reply:
Yeah you would say: 
```
join.object({
    someFeature: joi.boolean()
})
```

Reply:
that means every time we add a flag the schema needs updating though

Reply:
I was looking for something more like this https://joi.dev/api/?v=17.7.0#objectpatternpattern-schema-options